packages/medusa/src/commands/user.js

```diff
@@ -15,8 +23,17 @@ export default async function ({ directory, id, email, password, keepAlive }) {
       expressApp: app,
     })
 
-    const userService = container.resolve("userService")
-    await userService.create({ id, email }, password)
+    if (invite) {
+      const inviteService = container.resolve("inviteService")
+      await inviteService.create(email, "admin")
```

Comment:
suggestion: Handle user already exist error more elegantly with an explanatory server message: 
<img width="1118" alt="CleanShot 2023-06-05 at 11 10 27@2x" src="https://github.com/medusajs/medusa/assets/59018053/4a95c3d9-69c8-466e-919d-69ec7eed4a03">

Reply:
Sames goes for when `--invite` is not used

Reply:
Actually, this is not relevant for when `--invite` is passed. We already print a nice one here.

Reply:
I handled this in the `UserService`, let me know if I should move the handling into the command instead.